### ISSUE TITLE
<mask shape question> in English-to-Spanish translation with a sequence-to-sequence Transformer

### ISSUE THREAD

**Issue by moo611** [keras-team/keras-io#736]
I have a question in TransformerEncoder part. I found that MultiHeadAttention Layer receives a mask which shape is (B, T, S), but when I print the padding_mask from your code, I found the shape is (None,1,1,None). It's different.Is there something wrong?

**Comment by pcoet** [comment]
Hi @moo611, thanks for reaching out. Can you repost this question in one of the forums below? That way your question will have more visibility.

* [TensorFlow forum](https://discuss.tensorflow.org/).
* [Keras mailing list](https://groups.google.com/forum/#!forum/keras-users).

Thanks!
